packages/core/src/components/ModalNew/Modal/Modal.tsx

```diff
@@ -21,32 +24,48 @@ const Modal = forwardRef(
       onClose,
       children,
       className,
-      id,
       "data-testid": dataTestId
     }: ModalProps,
     ref: React.ForwardedRef<HTMLDivElement>
   ) => {
+    const [titleId, setTitleId] = useState<string>("");
+    const [descriptionId, setDescriptionId] = useState<string>("");
```

Comment:
If you dont give it a default value you wouldn't need the `undefined` in lines 56-57, no?

Reply:
it will set the attribute with empty string, I want it to not be shown at all
![image](https://github.com/user-attachments/assets/a97eca3f-3846-48a6-9d2d-b80fbcab7aae)
![image](https://github.com/user-attachments/assets/22478007-4227-4adf-bc3f-d05c065b6b55)

Reply:
I meant useState<string>()

Reply:
oh
good point, lol